package/mediacenter-addon-osmc/src/script.module.osmccommon/resources/lib/osmccommon/grablogs.py

```diff
@@ -920,31 +920,32 @@ def grab_log(self, ltyp, actn, name, key, hwid=''):
     def _mask_sensitive(self):
         # mask potentially sensitive information in blotter
 
+        re_to_mask_00 = re.compile(r'((?:OAuth|Bearer)\s)[^\'"]+')
+        re_to_mask_01 = re.compile(r'(["\']client_secret["\']:\s*[\'"])[^\'"]+')
+        re_to_mask_02 = re.compile(r'(client_secret=).+?(&|$|\|)')
+        re_to_mask_03 = re.compile(r'(["\'](?:nauth)*sig["\']: ["\'])[^\'"]+')
+        re_to_mask_04 = re.compile(r'(<[pP]ass(?:word)?>)[^<]+(</[pP]ass(?:word)?>)')
+        re_to_mask_05 = re.compile(r'(["\']password["\']:\s*[\'"])[^\'"]+')
+        re_to_mask_06 = re.compile(r'(password=).+?(&|$|\|)')
+        re_to_mask_07 = re.compile(r'(\w://.+?:).+?(@\w+)')
```

Comment:
I don't think this catches
`<setting id="services.webserverpassword" default="true">osmc</setting>`
for example, ie the 'new' method of using attributes rather than node names in guisettings.xml